frontends/mit-open/src/pages/FieldPage/FieldSearchFacetDisplay.tsx

```diff
@@ -96,22 +107,49 @@ const AvailableFacetsDropdowns: React.FC<
           constantSearchParams[facetSetting.name as FacetKey] || null,
         )
 
+        const isMultiple = BOOLEAN_FACET_NAMES.includes(facetSetting.name)
+          ? false
+          : true
+
+        let displayValue
+        if (BOOLEAN_FACET_NAMES.includes(facetSetting.name)) {
+          displayValue =
+            activeFacets[facetSetting.name as BooleanFacetKey] === true ||
+            activeFacets[facetSetting.name as BooleanFacetKey] === false
+              ? (
+                  activeFacets[facetSetting.name as BooleanFacetKey] as boolean
+                ).toString()
+              : ""
+        } else {
+          displayValue = activeFacets[facetSetting.name as FacetKey] || []
+        }
+
         return (
           facetItems.length > 0 && (
             <FormControl key={facetSetting.name}>
               <Select
-                multiple
+                multiple={isMultiple}
                 displayEmpty
-                value={activeFacets[facetSetting.name as FacetKey] || []}
+                value={displayValue}
                 renderValue={() => {
                   return facetSetting.title
                 }}
                 onChange={getHandleChangeForFacet(
-                  activeFacets[facetSetting.name as FacetKey] || [],
+                  activeFacets[facetSetting.name as FacetKey | BooleanFacetKey],
                   facetSetting.name,
                 )}
                 sx={{ m: 1, minWidth: 140 }}
               >
+                {!isMultiple ? (
+                  <MenuItem
+                    value=""
+                    key={facetSetting.name.concat(":", "unselect-all")}
```

Comment:
Minor naming nitpick: "unselect" seems better. It's not a multi-select, so you can't select more than one anyway.
```suggestion
                    key={facetSetting.name.concat(":", "unselect")}
```